src/InsertionsClient/Api/Props/Models/PropsFile.cs

```diff
@@ -85,9 +85,7 @@ public bool TryUpdateVariable(string variableName, string value, out string exis
 						}
 
 						isDirty = true;
-						textNode.Value = textValue.Substring(0, match.Index) +
-							variableName + "=" + value +
-							textValue.Substring(match.Index + match.Length);
+						textNode.Value = $"{textValue.Substring(0, match.Index)}{variableName}={value};{textValue.Substring(match.Index + match.Length)}";
```

Comment:
any chances the indexes can be out of range in this expression?  Would unit tests specifically for this string composition add value?

Reply:
I tested the cases where the match is at the very end, at the very beginning or somewhere in between. I haven't managed to get an exception. As long as the data in `match` object is valid, we should be fine.